catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardTypeImpl.java

```diff
@@ -54,6 +55,8 @@
   /** Set of {@link AttributeDescriptor}s */
   protected transient Set<AttributeDescriptor> descriptors = new HashSet<>();
 
+  protected transient HashMap<String, AttributeDescriptor> descriptorNameMap = new HashMap<>();
```

Comment:
✏️ Since we're only using regular `Map` methods on this reference, why not just leave it as `Map<String, AttributeDescriptor>`?

Reply:
oops. fixed that.